python/pfs/utils/dummyCableB.py

```diff
@@ -237,12 +237,12 @@ def makePfsDesign(pfsDesignId, fiberId):
     dec = np.zeros_like(fiberId, dtype=float)
     pfiNominal = np.zeros((num, 2), dtype=float)
 
-    fiberMags = [[] for _ in fiberId]
+    fluxes = [[] for _ in fiberId]
     filterNames = [[] for _ in fiberId]
 
     return PfsDesign(pfsDesignId, raBoresight, decBoresight,
                      fiberId, tract, patch, ra, dec, catId, objId, targetTypes, fiberStatus,
-                     fiberMags, filterNames, pfiNominal)
+                     fluxes, fluxes, fluxes, fluxes, fluxes, fluxes, filterNames, pfiNominal)
```

Comment:
Formerly the last 3 'fluxes' are actually flux errors. Add another dummy variable for that?